Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "skip-fail"
+version = "1.0.0"
+license = "MIT"
+authors = ["Jean SIMARD <woshilapin@tuziwo.info>"]
```

Comment:
I would rather put `Kisio Digital <team.coretools@kisio.org>`

Reply:
Indeed, I just created the project with `cargo new` and I didn't even touch some of these properties. But you proposition is much better!
Done in 92607094ab55e04e85457146ca6834b790aa1166.